shared/actions/chat.js

```diff
@@ -342,23 +384,66 @@ function * _setupNewChatHandler (): SagaGenerator<any, any> {
 const followingSelector = (state: TypedState) => state.config.following
 
 function * _loadInbox (): SagaGenerator<any, any> {
-  const inbox: GetInboxAndUnboxLocalRes = ((yield call(localGetInboxAndUnboxLocalRpcPromise, {param: {
-    query: {
-      status: Object.keys(CommonConversationStatus).filter(k => !['ignored', 'blocked'].includes(k)).map(k => CommonConversationStatus[k]),
+  const channelConfig = singleFixedChannelConfig([
+    'chat.1.chatUi.chatInboxUnverified',
+    'chat.1.chatUi.chatInboxConversation',
+    'chat.1.chatUi.chatInboxFailed',
+    'finished',
+  ])
+
+  const loadInboxChanMap: ChannelMap<any> = localGetInboxNonblockLocalRpcChannelMap(channelConfig, {
```

Comment:
Load the unverified inbox and deal with the stream of validations / errors. When the rpc is finished we clear out anything that didn't validate